apps/transport/lib/opendatasoft/url_extractor.ex

```diff
@@ -100,23 +100,23 @@ defmodule Opendatasoft.UrlExtractor do
  ## Examples
      iex> "name,file\\ntoulouse,http"
      ...> |> ImportData.get_url_from_csv()
-     {:ok, "http"}
+     {:ok, {url: "http", title: "http"}}
```

Comment:
I don't understand the examples, what is the url, what is the title in the csv ?

Reply:
By default, the title and url are the same.

However, we try to make a HEAD request to guess the actual filename. And this, my friend, is a bit tedious to test in doc"